src/common/components/table/table-examples/MempoolTable.tsx

```diff
@@ -0,0 +1,391 @@
+      css={{
+        '& > td:first-of-type': {
+          borderTopLeftRadius: 'redesign.md',
+          borderBottomLeftRadius: 'redesign.md',
+        },
+        '& > td:last-of-type': {
+          borderTopRightRadius: 'redesign.md',
+          borderBottomRightRadius: 'redesign.md',
+        },
+      }}
+      onClick={onClick}
+      cursor="pointer"
+      className="group"
+    >
+      <ChakraTable.Cell colSpan={numColumns} py={2} px={1}>
+        <Flex
+          alignItems="center"
+          justifyContent={{ base: 'flex-start', md: 'center' }}
+          gap={1.5}
+          boxShadow="0px 4px 4px 0px rgba(252, 100, 50, 0.25), 0px 4px 4px 0px rgba(255, 85, 18, 0.25)"
+          border="1px dashed var(--stacks-colors-accent-stacks-500)"
+          borderRadius="redesign.lg"
+          h={12}
+          px={4}
+        >
+          <Box display="inline-flex">
+            <Text fontSize="sm" fontWeight="medium" color="textSecondary">
+              New transactions have come in.
+            </Text>
+            &nbsp;
+            <Text
+              fontSize="sm"
+              fontWeight="medium"
+              color="textSecondary"
+              _groupHover={{ color: 'textPrimary' }}
+            >
+              Update list
+            </Text>
+          </Box>
+          <Icon h={3.5} w={3.5} color="iconTertiary" _groupHover={{ color: 'iconSecondary' }}>
+            <ArrowsClockwise />
+          </Icon>
+        </Flex>
+      </ChakraTable.Cell>
+    </ChakraTable.Row>
+  );
+};
+
+function getToAddress(tx: MempoolTransaction): string {
+  switch (tx.tx_type) {
+    case 'contract_call':
+      return tx.contract_call?.contract_id || '';
+    case 'token_transfer':
+      return tx.token_transfer?.recipient_address || '';
+    case 'smart_contract':
+      return tx.smart_contract?.contract_id || '';
+    default:
+      return '';
+  }
+}
+
+function getAmount(tx: MempoolTransaction): number {
+  switch (tx.tx_type) {
+    case 'token_transfer':
+      return parseInt(tx.token_transfer?.amount || '0');
+    case 'contract_call':
+      return 0;
+    default:
+      return 0;
+  }
+}
+
+export function MempoolTable({
+  columnDefinitions,
+  disablePagination = false,
+  sort = 'age',
+  order = 'desc',
+  filters,
+  initialData,
+}: {
+  columnDefinitions?: ColumnDef<MempoolTableData>[];
+  disablePagination?: boolean;
+  sort?: 'age' | 'size' | 'fee';
+  order?: 'asc' | 'desc';
+  filters?: Partial<TxTableFilters>;
+  initialData?: GenericResponseType<CompressedMempoolTxTableData>;
+}) {
+  const queryClient = useQueryClient();
+  const isCacheSetWithInitialData = useRef(false);
+
+  const [pagination, setPagination] = useState<PaginationState>({
+    pageIndex: 0,
+    pageSize: TX_TABLE_PAGE_SIZE,
+  });
+
+  const { fromAddress, toAddress, transactionType } = filters || {};
+
+  if (isCacheSetWithInitialData.current === false && initialData) {
+    const queryKey = ['mempoolTransactionsInfinite', sort, order];
```

Comment:
Isnt using filters or pagination state in teh query key

Reply:
mempool page has no pagination